src/pybullet_helpers/ikfast/utils.py

```diff
@@ -0,0 +1,316 @@
+# pylint: skip-file
```

Comment:
is this temporary? or do you think we'll need to keep it?

Reply:
Temporary so I could get the unit tests to run by telling pylint not to complain about this module